aspnetcore/client-side/angular/sample/AngularJSSample/src/AngularJSSample/Controllers/Api/PersonController.cs

```diff
@@ -16,7 +18,11 @@ public JsonResult GetPeople()
                 new Person { Id = 1, FirstName = "Bob", LastName = "Parker" }
             };
 
-            return Json(people);
+            // force the Json serializer to keep the case of the model properties
+            var serializerSettings = new JsonSerializerSettings();
+            serializerSettings.ContractResolver = new DefaultContractResolver();
+
```

Comment:
I recommend not modifying this controller at all. Instead, please modify the bindings in `Views/People/Index.cshtml` such that the casing matches the following:

```html
<h2>{{person.firstName}} {{person.lastName}}</h2>
```

This way, the casing of the property names is consistent with how they appear in other areas of the doc.